frontend/lib/src/components/widgets/ButtonGroup/ButtonGroup.tsx

```diff
@@ -91,21 +92,20 @@ function syncWithWidgetManager(
 
 function getContentElement(content: string): ReactElement {
   const fontSize = "lg"
-  if (isMaterialIcon(content)) {
-    return <DynamicIcon size={fontSize} iconValue={content} />
+  if (isFilledStarIcon(content)) {
+    return (
+      <StreamlitMarkdown
```

Comment:
nit: we might be able to directly render the `img` element without even going through markdown here (probably with some CSS props so that it aligns the same as with markdown). Not a big advantage, but might be slightly clenaer

Reply:
good point! I have updated it 👍